packit_service/worker/handlers/bodhi.py

```diff
@@ -122,6 +123,21 @@ def get_trigger_type_description(self, koji_build_data: KojiBuildData) -> str:
             retriggered Bodhi update
         """
 
+    @staticmethod
+    def get_handler_specific_task_accepted_message(
+        service_config: ServiceConfig,
+    ) -> str:
+        user = (
+            "packit" if service_config.deployment == Deployment.prod else "packit-stg"
+        )
+
+        return (
+            f"You can check the recent Bodhi update activity of `{user}` in "
+            f"[the Bodhi interface](https://bodhi.fedoraproject.org/users/{user}) (we "
+            f"have also planned adding support for viewing the updates"
+            f" in [Packit dashboard]({service_config.dashboard_url}))"
```

Comment:
Not sure if our users understand that we act as this FAS user.

```suggestion
            f"Your request has been accepted. "
            f"The Bodhi update will be created by `{user}` user and will appear in "
            f"[the Bodhi interface](https://bodhi.fedoraproject.org/users/{user})."
```

Also, I would either ignore the last sentence or link the respective issue.

Reply:
good point, added the link

Reply:
oh, missed the first note; I think they do, and to me this formulation is quite clear, but let me know if you think it should be changed